RHDEVDOCS-1817 + RHDEVDOCS-2454: Creating a VM via the +Add view of ODC and viewing VMs (#60992)
openshift/openshift-docs: modules/odc-creating-vms.adoc

// Module included in the following assemblies:
//
// * virt/virtual_machines/virt-create-vms.adoc

:_content-type: PROCEDURE
[id="odc-creating-vms_{context}"]
= Using the Developer perspective to create a virtual machine

You can create a virtual machine by using the *Developer* perspective of the {product-title} web console.

.Prerequisites

* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
* You have logged in to the {product-title} web console and are in the *Developer* perspective.

.Procedure

. In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.

. Select a virtual machine type from the *Template catalog*.

> . Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.

Author: Added a note 👍🏻

// Module included in the following assemblies:
//
// * virt/virtual_machines/virt-create-vms.adoc

:_content-type: PROCEDURE
[id="odc-creating-vms_{context}"]
= Using the Developer perspective to create a virtual machine

You can create a virtual machine by using the *Developer* perspective of the {product-title} web console.

.Prerequisites

* A cluster administrator has installed the {VirtProductName} Operator in your cluster.
* You have logged in to the {product-title} web console and are in the *Developer* perspective.

.Procedure

> . In the *+Add* view, click on the *Virtual Machines* tile to see the *Create new VirtualMachine* page.

Reviewer: ```suggestion
. In the *+Add* view, click the *Virtual Machines* tile to see the *Create new VirtualMachine* page.
```
Per ISG, do not use "click on". Use "click" instead.

. Select a virtual machine type from the *Template catalog*.

> . Click *Quick create VirtualMachine* to quick create a virtual machine, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.

Reviewer: ```suggestion
. Click *Quick create VirtualMachine* to create a virtual machine with no further input, or click *Customize VirtualMachine* to see the *Customize Template parameters* page.
```
- Not sure if this changes the meaning. Or maybe add `to create a virtual machine immediately`. Anyway, I don't think it should be the exact phrase from the button/link

Author: nice suggestion, thanks

+
[NOTE]
====
For Microsoft Windows virtual machines, only the *Customize VirtualMachine* option is available.
====

> . If you choose to customize your virtual machine, the *Customize Template parameters* page allows you to modify *Storage* and *Optional Parameters* settings. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.

Reviewer: ```suggestion
. If you choose to customize your virtual machine, you can modify *Storage* and *Optional Parameters* settings on the *Customize Template parameters* page. When you have finished customizing your virtual machine settings, click *Next* to see the *Review and create VirtualMachine* page. Review your settings, then click *Create VirtualMachine*.
```
- Avoid saying that inanimate things "allow" or "enable" humans to do tasks (anthropomorphism).
